### DISCUSSION TITLE
Missing function

### DISCUSSION THREAD

**Discussion by Super-Code774** [Magic-Fishes/Ecole-Directe-Plus#57] [General]
Missing fonction on the app "sanction" tab for sanctions see the info

**Comment by Super-Code774** [comment]
this function is also missing: send messages

**Comment by Truiteseche** [comment]
Hi (welcome) @Super-Code774,
Sorry I'm not sure I understand what you're talking about

**Reply by Super-Code774** [reply]
this function is also missing: send mail on ecoleDirect plus

**Reply by Super-Code774** [reply]
Sorry for answering late

**Reply by Super-Code774** [reply]
Sanction Tab: The tab to view sanctions is also missing. This feature is essential for monitoring sanctions.

**Reply by Truiteseche** [reply]
Indeed for the moment it is impossible to send messages from EDP. Consider opening an issue if you mind, but we're still not sure to implement it due to security and reliability reasons.
Regarding the second point, the https://github.com/Magic-Fishes/Ecole-Directe-Plus/pull/55 pull request was recently merged and may have solved the issue you mention, but it has not yet been deployed

**Reply by Super-Code774** [reply]
Ok thank
